ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/services/ApiSubmissionServiceSpec.scala

```diff
@@ -489,6 +489,7 @@ object ApiSubmissionServiceSpec {
       errorCodesVersionSwitcher = new ErrorCodesVersionSwitcher(
         enableSelfServiceErrorCodes = useSelfServiceErrorCodes
       ),
+      checkOverloaded = _ => None, // Used and tested in Canton
```

Comment:
Why not add a test here as well? I.e., `ApiSubmissionService` and how the 'overload checking' function is used is still defined in the Daml SDK so this feature should be tested here as well.